src/shared/codelens/pythonCodeLensProvider.ts

```diff
@@ -0,0 +1,122 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+import * as path from 'path'
+import * as vscode from 'vscode'
+
+import { DebugConfiguration } from '../../lambda/local/debugConfiguration'
+import { LambdaHandlerCandidate } from '../lambdaHandlerSearch'
+import { getLogger } from '../logger'
+import { DefaultSamCliProcessInvoker, DefaultSamCliTaskInvoker, } from '../sam/cli/samCliInvoker'
+import { getDebugPort } from '../utilities/vsCodeUtils'
+
+import {
+    CodeLensProviderParams,
+    getInvokeCmdKey,
+    makeCodeLenses
+} from './codeLensUtils'
+import { LambdaLocalInvokeArguments, LocalLambdaRunner } from './localLambdaRunner'
+
+export const PYTHON_LANGUAGE = 'python'
+export const PYTHON_ALLFILES: vscode.DocumentFilter[] = [
+    { language: PYTHON_LANGUAGE }
+]
+
+// TODO: Fix this! Implement a more robust/flexible solution. This is just a basic minimal proof of concept.
+const getSamProjectDirPathForFile = async (filepath: string): Promise<string> => {
+    return path.dirname(filepath)
+}
+
+export const getLambdaHandlerCandidates = async ({ uri }: { uri: vscode.Uri }): Promise<LambdaHandlerCandidate[]> => {
```

Comment:
function declaration style

Reply:
Made it non-exported.

Reply:
`export function getLambdaHandlerCandidates(...) { ... }`